scripts/composer/ScriptHandler.php

```diff
@@ -40,11 +40,9 @@ public static function createRequiredFiles(Event $event) {
       $fs->copy($drupalRoot . '/sites/default/default.settings.php', $drupalRoot . '/sites/default/settings.php');
       require_once $drupalRoot . '/core/includes/bootstrap.inc';
       require_once $drupalRoot . '/core/includes/install.inc';
-      $settings['config_directories'] = [
-        CONFIG_SYNC_DIRECTORY => (object) [
-          'value' => Path::makeRelative($drupalFinder->getComposerRoot() . '/config/sync', $drupalRoot),
-          'required' => TRUE,
-        ],
+      $settings['config_sync_directory'] = (object) [
```

Comment:
Not a blocker, but this section can be removed as it's overridden in the dockerfile - see https://github.com/UN-OCHA/slt8-site/pull/37#discussion_r678962876 (if it's needed for docksal development, let me know and I can replace it for SLT.)